### DISCUSSION TITLE
Adjust beatmap offset depending on the last 5 or 10 plays (just a suggestion)

### DISCUSSION THREAD

**Discussion by valflrt** [ppy/osu#20369] [Ideas] [answered]
Maybe adding the possibility to adjust beatmap offset depending on the last 5, 10 or even 20 last plays could make the offset more precise because the more data we have, the more accurate is the average. The "stats" of the last plays could be saved in a stack, when a new entry is added, the older entry is removed. By using the saved "stats" we could get a more accurate average. Also, maybe the stack should be cleared when resetting the offset so the saved "stats" don't affect the new offset average.

*(btw don't mind my english I'm not from an english-speaking country)*

**Comment by peppy** [comment, marked as answer]
One play should be enough, I think.

**Reply by MaxKruse** [reply]
In the same vein, would it be feasable to have a sort-of automatic offset adjustment? Sometimes the overlay on the left is not displayed long enough for me (e.g. when spamming to get back into the map, lack of familiarity with the feature possibly).

Perhaps as an option toggle

**Reply by peppy** [reply]
I have plans on improving the player loader screen to make it easier to make changes when you want to.
